go/bundle/cmd/sign-bundle/integrityblock.go

```diff
@@ -48,9 +48,7 @@ func obtainIntegrityBlock(bundleFile *os.File) (*integrityblock.IntegrityBlock,
 		return nil, 0, err
 	}
 
-	// Unlike web bundle length, integrity block length cannot be more than max int.
-	integrityBlockLen := int(fileStats.Size() - webBundleLen)
-
+	integrityBlockLen := fileStats.Size() - webBundleLen
```

Comment:
This can be negative if the WebBundle's last 8 bytes are corrupted. Should we have a guard against that?

Reply:
Added!